### PR TITLE
Scripts of database backups

### PR DESCRIPTION
**Pull request by savi-1311**
Solves Issue #9 
The cron job runs weekly to create a dump folder in the scripts folder with database backup.
The changes made-
1. Added scripts folder in the root directory
2. Created scripts/backup.js containing scripts for scheduled backup.
3. Added MONGOURI field in the example .env file.
4. Added require('corn') in server/index.js to run the script.

### CODE REVIEW

File: scripts/backup.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const CronJob = require('cron').CronJob;
+ const exec = require('child_process').exec;
+require('dotenv').config('../.env');
```
Comment (manishprivet) [inline scripts/backup.js:3]: The dotenv should be called before or into the entrypoint of the server which is index.js

It should either be in `server/index.js`, or better yet, in the`package.json`, set the start script to 

`node -r dotenv/config dist/index.js`

File: .env.example [hunk 2]
```diff
@@ -6,6 +6,8 @@ REQUEST_LIMIT=1mb
 OPENAPI_SPEC=/spec
 OPENAPI_ENABLE_RESPONSE_VALIDATION=false
 
+MONGOURI= mongodb://<username>:<password>@cluster0-shard-00-00-46c53.mongodb.net:27017,cluster0-shard-00-01-46c53.mongodb.net:27017,cluster0-shard-00-02-46c53.mongodb.net:27017/<dbname>?ssl=true&replicaSet=cluster0-shard-0&authSource=admin&retryWrites=true&w=majority
```
Comment (manishprivet) [inline .env.example:9]: This env seems unnecessary + you accidentally revealed your atlas cluster ID

**Review by manishprivet — CHANGES_REQUESTED** [review]

**Comment by GunaShekar02** [comment]
Hey @savi-1311, the scripts look good to go. But as I said, these scripts will run separately, independent of the main server.
Please reverse any changes made to the current server, and you can `npm init` in the scripts folder and start a new server there. Everything related to the scripts will be in the new server itself and that will function independent of the main server. 
So basically all of your script-related code is great, you just have to migrate it to a new server in a new folder(scripts folder) and reverse any changes made to the main server folder.

Marking this PR approved for now, hope to see the changes implemented!

**Review by GunaShekar02 — APPROVED** [review]

**Comment by savi-1311** [comment]
@GunaShekar02 Sir I have done the required changes.

**Pull request closed without merging**